### DISCUSSION TITLE
Detailed info about logged user. OIDC

### DISCUSSION THREAD

**Discussion by arturovivas** [authelia/authelia#5523] [Q&A] [answered]
Hi,

I just configured the Authelia as oidc to be used together with a Grafana instance. I have an external service that is being controlled by some UI Grafana, yet I would like to verify if those requests really come from an authorized and logged user according to the Authelia oidc. 

Grateful for any help how to implement a solution such that the external service is able to verify this using the Authelia oidc.

**Comment by james-d-elliott** [comment, marked as answer]
I don't believe there is any way for us to do that on the Authelia side unless the app in question makes the authroization request, I believe Grafana would have to allow forwarding the identity token for this purpose or use token exchange.
